src/hooks/useWallet.ts

```diff
@@ -36,31 +36,35 @@ export default function useWallet() {
     }
 
     const supportedClients = Object.keys(clients) as PROVIDER_ID[]
-
     setProviders(
-      supportedClients.map((id) => {
-        return {
-          ...allClients[id],
-          accounts: getAccountsByProvider(id),
-          isActive: activeAccount?.providerId === id,
-          isConnected: connectedAccounts.some((accounts) => accounts.providerId === id),
-          connect: () => connect(id),
-          disconnect: () => disconnect(id),
-          reconnect: () => reconnect(id),
-          setActiveProvider: () => setActive(id),
-          setActiveAccount: (account: string) => selectActiveAccount(id, account)
-        }
-      })
+      supportedClients
+        // Femove any clients that didn't initialise
+        .filter((id) => !!clients?.[id])
+        .map((id) => {
+          return {
+            ...allClients[id],
+            // Override static details with any instance details
+            ...clients[id],
```

Comment:
Added this line to add in any instance metadata (like name and icon in the custom wallet client)